task1.js

```diff
@@ -0,0 +1,121 @@
+class EventEmitter {
```

Comment:
Please, move EventEmitter to a separate file: you don't need to log all events from task1 for task2

Reply:
Done.